### ISSUE TITLE
PANIC on Mac OS

### ISSUE THREAD

**Issue by khalidchawtany** [neovim/neovim#16690]
### Neovim version (nvim -v)

NVIM v0.7.0-dev+732-g80210c189

### Vim (not Nvim) behaves the same?

no

### Operating system/version

MacOS 11.6

### Terminal name/version

iTerm 2 Build 3.4.14

### $TERM environment variable

xterm-256color

### Installation

brew install --HEAD neovim

### How to reproduce the issue

```nvim -u NONE```

### Expected behavior

Not to panic

### Actual behavior

Neovim panics.

```
PANIC: unprotected error in call to Lua API (bad argument #3 to '?' (function expected, got nil))
```

**Comment by clason** [comment]
duplicate of #16688

**Comment by clason** [comment]
It's an issue with homebrew's formula not tracking changes to HEAD; if you must use a prerelease version, clone and build from the Neovim repo.

**Comment by gpanders** [comment]
Should be fixed by #16714.
